public/index.html

```diff
@@ -1,86 +1,99 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
The HTML5 doctype declaration is typically written in uppercase as <!DOCTYPE html>. Both <!doctype html> and <!DOCTYPE html> are technically valid, but the uppercase version is more commonly used and recommended for consistency and readability.